lib/recurly/pricing/attachment.js

```diff
@@ -63,6 +63,7 @@ export default class Attachment {
     const updateAddon = elems.addon && updating('addon');
     const updateCurrency = updating('currency');
     const updateCoupon = elems.coupon && (updating('coupon') || updating('plan'));
+    const updateGiftcard = elems.giftcard && (updating('giftcard'));
```

Comment:
thinking we should use snake-casing on this field, similar to `postal_code`. 'addon' is somewhat strange because it is called 'addon' in the API despite being two words.